discover/portal/articles/130-forms/02-managing-form-entries.markdown

```diff
@@ -0,0 +1,94 @@
+# Managing Form Entries [](id=managing-form-entries)
+
+Once users begin submitting form entries, you'll want to do these things with
+them:
+
+- View form entries
+- Export form entries
+- Delete form entries
+
+Start by learning how to access and view the entries.
+
+## Viewing Form Entries [](id=viewing-form-entries)
+
+When users fill out forms, they're generating data. You're going to want to see
+it at some point. 
+
+1.  From the Menu, navigate back to the *Content* &rarr; *Forms* section of The
+    Lunar Resort site.
+
+2.  Click the *Actions* (![Actions](../../images/icon-actions.png)) button and
```

Comment:
Hey @rbohl,

In the Content > Forms page there are 2 actions buttons, check this below:

![screen shot 2018-05-15 at 11 18 39 am](https://user-images.githubusercontent.com/6729287/40062666-202ae29a-5832-11e8-952a-f32d1ce16015.png)

Therefore, maybe would be nice make clear to the user we're referring the actions button of the created Form. What do you think about it?